docs/src/content/en/docs/getting-started/installation.mdx

```diff
@@ -18,11 +18,15 @@ Mastra also supports other LLM providers. For a full list of supported models an
 - Node.js `v20.0` or higher
 - An API key from a supported [Model Provider](/docs/getting-started/model-providers)
 
+## Install using the `create mastra` CLI
```

Comment:
With this change the numbered steps are no longer under "Prerequisites".

I changed it to `create mastra` because that's the more common name.